main.py

```diff
@@ -0,0 +1,89 @@
+import json
+import os
+import requests
+import re
```

Comment:
Wrap it inside try block, ex:
```suggestion
`try:
  import json
  import os
  import requests
  import re
except ImportError:
    raise ImportError('<any message you want here>')`
```

Reply:
The only `import` that would make sense to wrap in a `try` block is `import request`, as other imports are modules from the standard library.

I would also say, that instead of doing this, it would be better to just specify dependencies in a `requirements.txt` or even better `pyproject.toml` file.